source/_components/camera.proxy.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: page
+title: "Camera Proxy"
+description: "Instructions how to integrate a camera proxy within Home Assistant."
+date: 2018-03-08 19:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Camera
+ha_release: 0.65
+---
+
+
+The `proxy` camera platform allows you to pass another camera's output through post-processing routines and generate a new camera with the post-processed output.
+
+The current post-processing supports resizing the image/MJPEG as well as limiting the maximum refresh rate.
+
+The current proxy capabilities are intended to reduce the camera bandwidth for slower internet connections.
+
+To enable this camera in your installation, you must first have an existing working camera configured in Home Assistant.  Next, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+camera:
+  - platform: proxy
+    entity_id: camera.<existingcamera>
+    max_stream_width: 360
+    max_image_width: 720
+```
+
+Configuration variables:
```

Comment:
I'm merging it now because I want 0.65 to get released. Please submit another PR migrating this list to use our `configuration` extension. [example](https://github.com/home-assistant/home-assistant.github.io/blame/next/source/_components/media_player.songpal.markdown#L40)